ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -17,8 +17,8 @@ import {
   getSelectedPrimaryPurposeItems
 } from 'app/utils/research-purpose';
 import {
-  AdminFederatedWorkspaceDetailsResponse,
-  CloudStorageTraffic, ListClusterResponse,
+  CloudStorageTraffic,
```

Comment:
@als364 FYI. I'm not sure if there are unit tests covering this page yet, but I made it compile.